addons/binding/org.openhab.binding.chromecast/README.md

```diff
@@ -12,7 +12,12 @@ This can be configured on the binding level:
 
 ## Supported Things
 
-The binding currently supports the "classic" Chromecast that is an HDMI dongle as well as the Chromecast Audio, which only does audio streaming and offers a headphone jack.
+| Things             | Description                                                                  | Thing Type |
+|--------------------|------------------------------------------------------------------------------|------------|
+| Classic Chromecast | The HDMI dongle type Chromecast                                              | chromecast |
```

Comment:
`Google Home`, `Google Home Mini` and `Google Home Max` are also supported as they are auto-discovered as `chromecast`

Reply:
@engineergreen can you take a look at @kubawolanin 's comment as well.

Reply:
I beliece @engineergreen already did, check  https://github.com/openhab/openhab2-addons/pull/3058/commits/71bb00ff08a3989b97ab2ba2563b505a1d2251dd#diff-8d75b964f954f65f11950d506642292fR17

Reply:
I was looking for a new row :-) Thanks!